### PR TITLE
fix(CostByRegion): apply chart data

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
- Please take it lightly!!
- Functions for converting is copied from cost dashboard widget. So you don't have to see those.

스크린샷 2023-01-05 오후 2 38 38

### CODE REVIEW

File: src/services/dashboards/widgets/cost-by-region/CostByRegion.vue [hunk 1]
```diff
@@ -82,23 +115,74 @@ const state = reactive({
...
+            color: storeState.providers[provider]?.color || '',
+            provider,
+            value: cost as number,
+            pieSettings: {
+                fill: storeState.providers[provider]?.color || '',
+                stroke: storeState.providers[provider]?.color || '',
+            },
+        })),
+    }));
+};
```
Comment (yuda110) [inline src/services/dashboards/widgets/cost-by-region/CostByRegion.vue:178]: description: you don't have to see these codes.

File: src/services/dashboards/widgets/cost-by-region/CostByRegion.vue [hunk 2]
```diff
@@ -189,16 +295,38 @@ defineExpose<WidgetExpose<FullData>>({
...
+                        @apply inline-block rounded-full;
+                        margin-right: 0.25rem;
+                        width: 0.5rem;
+                        height: 0.5rem;
+                    }
+                    .label {
+                        @apply mr-4 text-gray-500;
+                        font-size: 0.75rem;
+                        line-height: 1.5;
+                        white-space: nowrap;
```
Comment (song2kim) [inline src/services/dashboards/widgets/cost-by-region/CostByRegion.vue:320]:
```
.label {
  @apply mr-4 text-label-sm text-gray-700;
  white-space: nowrap;
}
```
How about code like the one above according to the new typo convention (`text-label-sm`)?

**Review by song2kim — APPROVED** [review]
LGTM~! I left small comments

**Review by sulmoJ — APPROVED** [review]
LGTM!
speeeeeeedahyun~~~~~

File: src/services/dashboards/widgets/cost-by-region/CostByRegion.vue (hunk 2 above)
Comment (yuda110) [reply src/services/dashboards/widgets/cost-by-region/CostByRegion.vue:320]: oh I'll update this later commit!!

**Pull request merged**